lib/screens/story_screen.dart

```diff
@@ -9,61 +9,55 @@ import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
 import 'package:andax/models/story.dart';
 
-class StoryScreen extends StatefulWidget {
+class StoryScreen extends StatelessWidget {
   const StoryScreen(
     this.info, {
     Key? key,
   }) : super(key: key);
 
   final StoryInfo info;
 
-  @override
-  State<StoryScreen> createState() => _StoryScreenState();
-}
-
-class _StoryScreenState extends State<StoryScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         leading: const RoundedBackButton(),
-        title: Text(widget.info.title),
+        title: Text(info.title),
       ),
       body: ListView(
         padding: const EdgeInsets.only(bottom: 76),
         children: [
-          if (widget.info.description != null)
+          if (info.description != null)
             ListTile(
               leading: const Icon(Icons.info_rounded),
               title: Text(
-                widget.info.description!,
+                info.description!,
               ),
             ),
         ],
       ),
       floatingActionButton: FloatingActionButton.extended(
         onPressed: () async {
-          final data = await showLoadingDialog<List>(
+          late final Story story;
+          late final Translation translation;
+          await showLoadingDialog<void>(
             context,
-            Future.wait([
-              loadStory(widget.info),
-              loadTranslation(widget.info),
-            ]),
+            (() async {
+              story = await loadStory(info);
+              translation = await loadTranslation(info);
+            })(),
+          );
```

Comment:
This feels overly complicated. What's wrong with simply
```dart
final Story story = await loadStory(info);
final Translation translation = await loadTranslation(info);

await showLoadingDialog<void>(
  context,
  Future.value(),
);
```
?

Reply:
This will not show dialog during loading, only for a moment right after it's loaded.

Reply:
Does it take that long to load the story and its translation? Also, isn't the "uninitialized late variable" problem we had yesterday related to this?